src/sentry/buffer/base.py

```diff
@@ -123,7 +123,7 @@ def incr(
         """
         process_incr.apply_async(
             kwargs={
-                "model": model,
+                "model_name": f"{model._meta.app_label}.{model._meta.model_name}",
```

Comment:
Do we need to roll this out by scheduling the task with both values?

Reply:
I think it's ok. The existing code handles both `model` and `model_name`. The only thing that can go wrong is that old tasks might be sending `model` and we'll see some info logs until the rollout finishes.

Reply:
The tasks already in production should handle model/model_name. This log message should only fire for a short period of time after this is deployed. If it continues afterwards, I've missed a buffer.incr() call.